mrmustard/lab_dev/circuits.py

```diff
@@ -461,26 +306,22 @@ def deserialize(cls, data: dict) -> Circuit:
         return circ
```

Comment:
on the lines right above this, I would just remove the `if path:` condition, and always update it - before this PR, it might have been `None`, but now this conditional checking is pointless.

Reply:
done